lib/logstash/plugin_mixins/http_client.rb

```diff
@@ -39,7 +39,11 @@ def setup_http_client_config
     # How many times should the client retry a failing URL? Default is `0`
     config :automatic_retries, :validate => :number, :default => 0
 
-    # If you need to use a custom X.509 CA (`.pem` certs) specify the path to that here
+    # Set this to false to disable SSL/TLS certificate validation
+    # Note: setting this to false is generally considered insecure!
+    config :verify_cert, :validate => :boolean, :default => true
```

Comment:
Thoughts about naming this `:ssl_certificate_verification`. Just so we can be consistent with `es-output` https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/lib/logstash/outputs/elasticsearch.rb#L282